src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -4359,7 +4359,6 @@
     </Field>
     <Field Name="Type" Type="TypeSyntax" Optional="true" Override="true"/>
     <Field Name="Identifier" Type="SyntaxToken" Optional="true">
-      <!-- Tracked by https://github.com/dotnet/roslyn/issues/78961 : enforce that a ParameterSyntax cannot be created with both Type and Identifier missing -->
```

Comment:
:memo" I prototyped such a change, but noticed that our generated syntax testing infrastructure currently creates worse `ParameterSyntax` nodes:
```
        private static ParameterSyntax GenerateParameter()
            => SyntaxFactory.Parameter(new SyntaxList<AttributeListSyntax>(), new SyntaxTokenList(), default(TypeSyntax), default(SyntaxToken), default(EqualsValueClauseSyntax));
```
I don't think it makes sense to add specific validation for Type and Identifier missing when we have a complete lack of validation as baseline. We could file an MQ issue for broader problem.